packages/database/src/core/DatabaseChangeChannel.js

```diff
@@ -1,18 +1,22 @@
-import PGPubSub from 'pg-pubsub';
-import winston from 'winston';
+import { generateId } from './utilities/generateId';
 
-import { generateId } from '../core/utilities/generateId';
-import { getConnectionConfig } from './getConnectionConfig';
-
-export class DatabaseChangeChannel extends PGPubSub {
-  constructor() {
-    super(getConnectionConfig(), { log: winston.info });
+export class DatabaseChangeChannel {
+  constructor(changeChannelAdapter) {
+    this.changeChannelAdapter = changeChannelAdapter;
```

Comment:
DatabaseChangeChannel now receives the adapter, either TupaiaChangeChannel or BrowserChangeChannel in order to configured pubsub differently